src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -49,9 +49,9 @@ enum {
 };
 
 typedef struct {
-    uint8_t         flags;
-    uct_ib_uint24_t qp_num;
-    uct_ib_uint24_t flush_rkey;
+    uct_rc_mlx5_base_ep_address_t super;
+    uint8_t                       flags;
+    uct_ib_uint24_t               flush_rkey;
```

Comment:
it looks like braking of backward compatibility...

Reply:
is there currently a scenario where gga transport is selected by default by UCP?

Reply:
I think it can be selected for `rma_bw` lane